changelog/db.changelog-1.11.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+  <changeSet author="Kirsten Williamson" id="1">
+    <tagDatabase tag="v1.8.0" />
+  </changeSet>
+  <changeSet author="Kirsten Williamson" id="2">
+    <insert tableName="schemes">
+      <column name="schemeId" value="8" />
+      <column name="sourceSystem" value="Manual" />
+      <column name="name" value="MANUAL" />
```

Comment:
Suggest changing `name` to title case, eg. `Manual`.  Then it's consistent with `Vet Visits` and `Lump Sums`.  Only the ones that are initials need to be caps.